load_testing/common_flows/flow_helper.py

```diff
@@ -16,10 +17,11 @@ def do_request(
         catch_response=True,
         name=name,
     ) as resp:
+        resp.raise_for_status()
+
         if expected_redirect:
             verify_resp_url(expected_redirect, resp)
 
-        resp.raise_for_status()
```

Comment:
I moved the `raise_for_status`, above the `expected_redirect` check.  I think this might have been hiding some status errors.